RIOT-OS/RIOT
# Doc rust maintenance

### Contribution description

This extends documentation around Rust to establish procedures for how to address Rust related breakage without involving third parties (like with releases to crates.io).

### Testing procedure

I hope we avoid ever testing the local-forks strategy, because that should IMO only come to bear if either a breaking change manages to go to master again (happens occasionally, but affecting the Rust side should be rare and only happened initially because a PR raced against the Rust-establishing PR), or when making crates.io releases becomes impossible on short term (and I hope that as Rust gains popularity, more maintainers will join maintaining the Rust side).

### Issues/PRs references

Closes: https://github.com/RIOT-OS/RIOT/issues/17488

(Not on its own, but with our Maintainers group added to the crates.io owners side, the bus factor aspect should already be addressed too).
